ebml_matroska.xml

```diff
@@ -308,28 +308,28 @@ this can be used for tracks containing only translations of foreign-language aud
+  <element name="FlagHearingImpaired" path="\Segment\Tracks\TrackEntry\FlagHearingImpaired" id="0x55AB" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp. 0 if that track is suitable resp. unsuitable for users with hearing impairments.</documentation>
     <extension webm="0"/>
     <extension divx="0"/>
   </element>
-  <element name="FlagVisualImpaired" path="\Segment\Tracks\TrackEntry\FlagVisualImpaired" id="0x55AC" type="uinteger" range="0-1" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track is suitable for users with visual impairments. (1 bit)</documentation>
+  <element name="FlagVisualImpaired" path="\Segment\Tracks\TrackEntry\FlagVisualImpaired" id="0x55AC" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp. 0 if that track is suitable resp. unsuitable for users with visual impairments.</documentation>
     <extension webm="0"/>
     <extension divx="0"/>
   </element>
-  <element name="FlagTextDescriptions" path="\Segment\Tracks\TrackEntry\FlagTextDescriptions" id="0x55AD" type="uinteger" range="0-1" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track contains textual descriptions of video content. (1 bit)</documentation>
+  <element name="FlagTextDescriptions" path="\Segment\Tracks\TrackEntry\FlagTextDescriptions" id="0x55AD" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp. 0 if that track contains resp. does not contain textual descriptions of video content.</documentation>
     <extension webm="0"/>
     <extension divx="0"/>
   </element>
-  <element name="FlagOriginal" path="\Segment\Tracks\TrackEntry\FlagOriginal" id="0x55AE" type="uinteger" range="0-1" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track is in the content's original language (not a translation). (1 bit)</documentation>
+  <element name="FlagOriginal" path="\Segment\Tracks\TrackEntry\FlagOriginal" id="0x55AE" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp. 0 if that track is in the content's original language resp. is a translation.</documentation>
     <extension webm="0"/>
     <extension divx="0"/>
   </element>
-  <element name="FlagCommentary" path="\Segment\Tracks\TrackEntry\FlagCommentary" id="0x55AF" type="uinteger" range="0-1" default="0" maxOccurs="1">
-    <documentation lang="en" purpose="definition">Set if that track contains commentary. (1 bit)</documentation>
+  <element name="FlagCommentary" path="\Segment\Tracks\TrackEntry\FlagCommentary" id="0x55AF" type="uinteger" range="0-1" maxOccurs="1">
+    <documentation lang="en" purpose="definition">Set to 1 resp 0 if that track contains resp. does not contain commentary.</documentation>
```

Comment:
here is resp written without a dot.
In all other cases you use "resp."

Reply:
I just force-pushed a fix for this. Thanks.